platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/TransactionSubmitter.java

```diff
@@ -365,6 +365,22 @@ public boolean canSubmitMore(Platform platform) {
                 return true;
             }
         } else if (this.goal == SUBMIT_GOAL.TRANS_PER_SECOND_PER_NODE) {
+            // if the node is offline for a while due to network disruption,
+            // it may try to catch up the lost transactions count by submitting
+            // many transactions in short window and lead to a burst of transactions. This is not good for
+            // platform, so we need to check the current TPS and make sure it is not too high.
+            final double tarnSubTSP = (double) metrics.getValue("Debug.info", "tranSubTPS");
+            if (tarnSubTSP > tranPerSecondGoal * 1.3) {
+                //                logger.info(
```

Comment:
can you remove the commented code or maybe define it as debug logging?

Reply:
Fixed